broker-core/src/main/java/io/zeebe/broker/workflow/processor/flownode/TerminateFlowNodeHandler.java

```diff
@@ -19,25 +19,24 @@
 
 import io.zeebe.broker.incident.data.IncidentRecord;
 import io.zeebe.broker.incident.processor.IncidentState;
-import io.zeebe.broker.workflow.model.element.ExecutableFlowElement;
+import io.zeebe.broker.workflow.model.element.ExecutableFlowNode;
 import io.zeebe.broker.workflow.processor.BpmnStepContext;
 import io.zeebe.broker.workflow.processor.BpmnStepHandler;
 import io.zeebe.broker.workflow.processor.EventOutput;
 import io.zeebe.broker.workflow.state.ElementInstance;
 import io.zeebe.protocol.intent.WorkflowInstanceIntent;
 
-public class TerminateFlowNodeHandler<T extends ExecutableFlowElement>
```

Comment:
The main idea behind having a type parameter here, was if you need to subclass this (which we do), it's sometimes useful to have the context with the correct element (to avoid casting). We don't need it anymore it seems, but I can imagine in the future where it could useful. It's arguable if this is the best approach anyway though, so I'm ok with removing it and figuring out something better when we need it.